### ISSUE TITLE
Construct query in debug view

### ISSUE THREAD

**Issue by tomas-knap** [mff-uk/odcs#165]
The current query dialog looks like:

![screen shot 2013-07-22 at 4 16 57 pm](https://f.cloud.github.com/assets/3014917/835367/ae6d0844-f2d9-11e2-9074-9158a7b3bc36.png)

It is really confusing.

Problem: 
1. Combo box with TTL, Trix, does not make any sense for SELECT queries. It has to be available only for CONSTRUCT queries. Solution: It should be clearly stated that combo box is only for construct queries by having label "Format for construct queries" (and  it should be in the right part of the panel)
1. When using CONSTRUCT query, the download should be immediate, no off-style button "download data.", which remains there between consequent queries, so that it is not clear new file was created. Solution: remove download button, immediate download.

**Comment by bogo777** [comment]
There is no problem with changing the label by Format combo box, but for immediate download I found only one solution in Vaadin, and it is no longer working with version 7 and recomended solution is how we have it now. I will look into it some more, but it will propably take some time.

**Comment by tomas-knap** [comment]
Ok, 

The main problem of the current solution is that: 
- if download link occurs, it remains there forever until any select query is executed
- when you click on Query for the second time you do not know that the link was regenerated.

So If you find solution for those two things, it is enough.

**Comment by bogo777** [comment]
I added icon to download link and changed label to display time on which data were queried. I also added notification after clicking on query button.

I found possible solution for immediate download:
https://vaadin.com/wiki/-/wiki/Main/Letting%20the%20user%20download%20a%20file

But it would possibly require special button for CONSTRUCT queries. I thought that maybe we could move SELECT querying to browse tab as hideable component, with prefilled default query for triples.

Confirming funcionality of immediate donwload and possible changes in Query/Browse tabs will be quite a lot of work, so I would postpone this to next iteration and for now just updated the current solution, to make it more clear that data were requeried?
